tests/performance/mixed_tensors/mixed_tensors.rb

```diff
@@ -0,0 +1,84 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+require 'performance_test'
+require 'app_generator/search_app'
+require 'performance/fbench'
+require 'pp'
+require 'environment'
+
+class MixedTensorPerfTest < PerformanceTest
+
+  FBENCH_RUNTIME = 30
+  LABEL = "label"
+  MIXED_BASIC = "mixed_basic"
+
+  def initialize(*args)
+    super(*args)
+  end
+
+  def test_mixed_tensor_operations
+    set_description("Test performance of various mixed tensor operations")
+    set_owner("geirst")
+    @graphs = get_graphs
+    deploy_and_prepare_data
+    feed_docs(5000)
+    run_fbench_helper
+  end
+
+  def deploy_and_prepare_data
+    deploy_app(create_app)
+    start
+    @container = vespa.container.values.first
+    compile_data_gen
+    gen_and_copy_query_file(100)
+  end
+
+  def create_app
+    SearchApp.new.sd(selfdir + "test.sd").
+      search_dir(selfdir + "search")
+  end
+
+  def compile_data_gen
+    @data_gen = dirs.tmpdir + "data_gen"
+    @container.execute("g++ -Wl,-rpath,#{Environment.instance.vespa_home}/lib64/ -g -O3 -o #{@data_gen} #{selfdir}/data_gen.cpp")
+  end
+
+  def gen_and_copy_query_file(num_queries)
+    @query_file = dirs.tmpdir + "queries.txt"
+    @container.execute("#{@data_gen} queries #{num_queries} > #{@query_file}")
+    @container.copy(@query_file, File.dirname(@query_file))
```

Comment:
this looks un-needed and wrong, since you generated the file on the container node, but "copy" assumes it was generated locally and should be copied to the container node.

Reply:
Yes, I think you are right. Nice catch. Let's try to remove this line.

Reply:
Fixed.